src-tauri/identity_wallet/src/state/actions/cancel_user_journey.rs

```diff
@@ -0,0 +1,17 @@
+use crate::reducer;
+use crate::state::actions::ActionTrait;
+use crate::state::actions::Reducer;
+use crate::state::reducers::cancel_user_journey;
+
+use ts_rs::TS;
+
+#[derive(serde::Serialize, serde::Deserialize, Debug, TS, Clone)]
+#[ts(export, export_to = "bindings/actions/CancelUserJourney.ts")]
```

Comment:
All exports by `ts_rs` that produce something like `export type Foobar = null;` can be removed.